docs/markdown/Compiler-properties.md

```diff
@@ -29,9 +29,16 @@ Compiler id
 ==
 
 The compiler object has a method called `get_id`, which returns a
-lower case string describing the "family" of the compiler. See
-[reference tables](Reference-tables.md) for a list of supported
-compiler ids.
+lower case string describing the "family" of the compiler.
+
+The compiler object also has a method `get_argument_syntax` which 
+returns a lower case string of `gcc`, `msvc`, or another undefined string
+value; identifying whether the compiler arguments use the same syntax as
+either `gcc` or `msvc`, or that it's arguments are not like either. This should
+only be used to select the syntax of the arguments to test.
```

Comment:
'to test'? ... with `has_argument()` presumably?

I like this, but I'm not sure we should be quite that restrictive:  There are instances in test cases where we (1) add arguments we 'know' are valid for msvc, or (2) build compilation commands entire.

Reply:
I've tried to clarify and soften that a little bit.